vic/drivers/shared_image/src/get_global_domain.c

```diff
@@ -31,7 +31,8 @@
  *****************************************************************************/
 size_t
 get_global_domain(char          *nc_name,
-                  domain_struct *global_domain)
+                  domain_struct *global_domain,
```

Comment:
This is a question in general about functions in c, but is it ever done to add explanations about what each of the inputs to a function do? I ask this because when I get down to the bottom and see line 77 of `vic_start.c` and see the addition of the `false` parameter I don't really know what that parameter is used for and I have to go back to this function and work though what the `coords_only` bool does. Is there a way to add an explanation of what each of the parameters is used for and should be?